reflex/.templates/apps/default/default.py

```diff
@@ -15,6 +15,7 @@ class State(rx.State):
 
 def index() -> rx.Component:
     return rx.fragment(
+        rx.connection_modal(),  # displays error when backend cannot connect
```

Comment:
Should we enable this in all apps by default? And have them specify if they want this in the config?